math-magicians/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] Your readme doesn't provide enough details about the project. Please remove the boilerplate from your Readme and add some info about your project. The Readme file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.

Reply:
@medaminedev66 
cool thanks for this advice i will move the files

Reply:
dear @medaminedev66 
the readme file will be updated when we merge this branch to the main branch 
thanks